packages/interfaces/src/waku.ts

```diff
@@ -6,10 +6,30 @@ import type { IFilter } from "./filter.js";
 import type { IHealthIndicator } from "./health_indicator.js";
 import type { Libp2p } from "./libp2p.js";
 import type { ILightPush } from "./light_push.js";
+import { IDecodedMessage, IDecoder, IEncoder } from "./message.js";
 import type { Protocols } from "./protocols.js";
 import type { IRelay } from "./relay.js";
 import type { IStore } from "./store.js";
 
+type AutoShardSingle = {
+  clusterId: number;
+  shardsUnderCluster: number;
+};
+
+type StaticShardSingle = {
```

Comment:
We removed `SingleShardInfo` a while ago -- is this related? Do we still need it?